kratos/tests/test_levelset_convection.py

```diff
@@ -175,23 +169,18 @@ def test_levelset_convection_BFECC_algebraic(self):
         linear_solver = linear_solver_factory.ConstructSolver(
             KratosMultiphysics.Parameters("""{"solver_type" : "skyline_lu_factorization"}"""))
 
-        model_part.CloneTimeStep(1.0)
+        model_part.CloneTimeStep(10.0)
 
         KratosMultiphysics.FindGlobalNodalNeighboursProcess(model_part).Execute()
 
-        KratosMultiphysics.ComputeNonHistoricalNodalGradientProcess(
-            model_part,
-            KratosMultiphysics.DISTANCE,
-            KratosMultiphysics.DISTANCE_GRADIENT,
-            KratosMultiphysics.NODAL_AREA).Execute()
-
         levelset_convection_settings = KratosMultiphysics.Parameters("""{
-            "max_CFL" : 1.0,
+            "max_CFL" : 0.2,
             "max_substeps" : 0,
-            "eulerian_error_compensation" : false,
+            "eulerian_error_compensation" : true,
             "element_type" : "levelset_convection_algebraic_stabilization",
             "element_settings" : {
-                "include_anti_diffusivity_terms" : true
+                "include_anti_diffusivity_terms" : true,
+                "requires_distance_gradient" : true

```

Comment:
```suggestion
                "include_anti_diffusivity_terms" : true
```
This should not be exposed.